resources/js/package.js

```diff
@@ -97,6 +97,10 @@ function init() {
         computed: {
             // Wrap the local storage in getter and setter functions so you do not have to interact using .value
             guestEmail: wrapValue(useLocalStorage('email', (window.debug ? 'wayne@enterprises.com' : ''), {serializer: StorageSerializers.string})),
+
+            loggedIn() {
+                return Boolean(this.$root.user?.id)
+            },
```

Comment:
```suggestion
            loggedIn: Boolean(this.$root.user?.id),
```

This doesn't work?

Reply:
Getting rid of the $root works, but it has to stay in its own function with the return unfortunately.